examples/js/effects/OutlineEffect.js

```diff
@@ -502,9 +502,9 @@ THREE.OutlineEffect = function ( renderer, parameters ) {
 
 	};
 
-	this.setScissorTest = function ( boolean ) {
+	this.setScissorTest = function ( enable ) {
```

Comment:
`scissorTest` instead of `enable`.

Reply:
That is not less generic in any sense of the word. "scissorTest" would imply that one could pass an actual test whereas passing true actually enables the test, doesn't it?

Reply:
It's just the way these parameters are defined in other parts of the library. `scissorTest` would be more consistent...

Reply:
I changed it as requested.